### ISSUE TITLE
Wraith Wallet 4.01 will not open on my macOS - High Sierra - Help Please

### ISSUE THREAD

**Issue by JMPClass** [vergecurrency/VERGE#482]
# Issue Template

## Prerequisites

* [x] Are you running the latest version?

## User Story:

* "I'm submitting a ... "
  - [ ] bug report
  - [ ] feature request
  - [ ] support request 
  - [ ] translation request 

## Steps to Reproduce (for bugs)

* Steps :
1.
2.
3.
4.

## Your Environment

* Verge Release:
* CPU:
* GPU:
* RAM:
* OS:
    - [ ] I'm using Mac
    - [ ] I'm using Linux
    - [ ] I'm using Mac

**Comment by marpme** [comment]
Is this the latest release or self-built from master branch?

**Comment by Max-Meldrum** [comment]
It would be helpful with more information. What exactly is happening? You can't open the program? Is it not syncing?

Is there any error message popping up?

**Comment by JMPClass** [comment]
Dear Max,

Absolutely,

OS:  macOS HS
Issue:  After a completely fresh install, when I try to open the program, the OS flashes a bug report. See Screen shot.
I was operating QT 3.1 without issue. 4.0.1 does not work.Thanks

![screen shot 2018-01-15 at 4 21 20 pm](https://user-images.githubusercontent.com/29355484/34941187-5307fd18-fa10-11e7-8b58-5c76019cadf2.png)

**Comment by marpme** [comment]
@JMPClass You have to install the boost library on mac first before starting the program.

Enter your terminal and enter: `brew install boost@1.66`
-> [Homebrew installation](https://brew.sh/index_de.html)

**Comment by JMPClass** [comment]
Marpme,

I installed Brew as directed.  Still having issues - see screen shots.  Thanks

JP
![1 2018-01-15 at 6 02 54 pm](https://user-images.githubusercontent.com/29355484/34944833-c06c3e60-fa1e-11e7-9c02-27ea0a2b896f.png)
![2 2018-01-15 at 6 03 17 pm](https://user-images.githubusercontent.com/29355484/34944849-cb2d13d8-fa1e-11e7-9f70-9ed2ec117e02.png)
![3 2018-01-15 at 6 03 41 pm](https://user-images.githubusercontent.com/29355484/34944878-dfca1f20-fa1e-11e7-9732-f4a805c73829.png)
![4 2018-01-15 at 6 04 04 pm](https://user-images.githubusercontent.com/29355484/34944887-ecab6ae6-fa1e-11e7-9e25-246593480517.png)
![5 2018-01-15 at 6 04 20 pm](https://user-images.githubusercontent.com/29355484/34944903-f9e12fca-fa1e-11e7-8734-8b1ffc56f864.png)

**Comment by JMPClass** [comment]
Gents,

Standby - I may not have installed the correct version - doing it again.

JP

**Comment by JMPClass** [comment]
Gents,

Installed 1.66 as directed - says my boost is not linked and wraith will still not open correctly.  Hmmm

Please assist - thanks

JP

**Comment by MrASquare** [comment]
Type this into your terminal:

xcode-select --install
ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
brew doctor
brew install boost

**Comment by ekryski** [comment]
Duplicate of #442

**Comment by JMPClass** [comment]
Mr.ASquare, et al.,

I have tried everything.  Still no luck.  

I installed the following:

xcode-select --install
ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
brew doctor
brew install boost

It now states that I have to link everything with Link-Boost?

I installed the Wraith Wallet on my Windows Side - works just fine.  Maybe I will just use that.  If anyone can help me to get this working on my macOS HS side, that would be great.  Please see latest screenshots
![screen shot 2018-01-16 at 11 01 19 am](https://user-images.githubusercontent.com/29355484/34974847-a7e0e948-faac-11e7-96ec-e914a84248eb.png)
.  Thanks.

JP

**Comment by JMPClass** [comment]
Gents,

Another screenshot
![screen shot 2018-01-16 at 11 10 03 am](https://user-images.githubusercontent.com/29355484/34975118-e6b61386-faad-11e7-8c64-313cbfc45759.png)
 of my issue.

Thanks

**Comment by sossienl** [comment]
It's not only you. Unfortunately this is getting painful. I tried on several occassions, with the commands. But no positive result

**Comment by Positivism** [comment]
brew link boost

**Comment by ebrinkster** [comment]
Try verge 4.0.1 fork for stealth.  No further issues with macOS for me after this release.
